feature/networkinstance/otg_tests/defaults_test/defaults_test.go

```diff
@@ -198,7 +210,7 @@ func TestDefaultAddressFamilies(t *testing.T) {
 				}
 				loss := tx - rx
 				lossPct := loss * 100 / tx
-				if got := lossPct; got > 0 {
+				if got := lossPct; got > float32(lossTolerancePct) {
```

Comment:
This change is omitted in the PR description, but I don't think any packet loss should be expected in this scenario.. can you elaborate?

Reply:
This is needed until ixia adds explicit ARP in VerifyARP. This should not affect physical platforms.